### DISCUSSION TITLE
What should I do if the panel does not load? در صورت لود نشدن پنل چه کنم؟

### DISCUSSION THREAD

**Discussion by TheTraxex** [hiddify/Hiddify-Manager#121] [Q&A سوالات رایج] [answered]
در صورت لود نشدن پنل چه کنم؟

**Comment by TheTraxex** [comment, marked as answer]
ابتدا سیستم رو ری‌استارت کنید. اگر همچنان با مشکل روبرو بودید،
```
sudo /opt/hiddify-config/install.sh
```
رو اجرا کنید. اگر همچنان مشکل پابرجا بود، فایل لاگ رو به ایمیل hiddify@gmail.com ارسال کنین تا بررسی کنیم.

**Reply by amirhmafi2000** [reply]
سلام تو ویندوز ۱۰ برنامه رو نصب کردم وقتی میخوام برنامه رو باز کنم (فایل exe) برنامه یه لحظه باز میشه و بسته میشه 
مشکل از کجاست

**Reply by LordRasooL** [reply]
همین مشکلو منم دارم
